lib/logger.js

```diff
@@ -648,7 +656,7 @@ class Logger extends EventEmitter {
         config.data = payload
       }
 
-      axios(config)
+      axios.request(config)
```

Comment:
whats the distinction here, just a more explicit alias?

Reply:
Yes.  At one point, people said that if you didn't do that, it would ignore `url` in the config.